deltachat-rpc-client/src/deltachat_rpc_client/account.py

```diff
@@ -0,0 +1,79 @@
+from typing import List, Optional
+
+from .chat import Chat
+from .contact import Contact
+from .message import Message
+
+
+class Account:
+    def __init__(self, rpc, account_id) -> None:
+        self.rpc = rpc
```

Comment:
Should this not be made private and be `self._rpc`?  Exposing this directly seems a bit too leaky?  It makes the API of the rpc part of the API of this class.

(applies to pretty much all classes, not just this one).